src/test/kotlin/org/rust/lang/core/completion/RsDeriveCompletionProviderTest.kt

```diff
@@ -128,4 +128,16 @@ class RsDeriveCompletionProviderTest : RsCompletionTestBase() {
         #[derive(std::marker::Clo/*caret*/)]
         struct S;
     """)
+
+    fun `test complete in cfg_attr`() = doSingleCompletion("""
+        #[cfg_attr(windows, derive(Debu/*caret*/))]
+        struct Test {
+            foo: u8
+        }
+    """, """
+        #[cfg_attr(windows, derive(Debug/*caret*/))]
+        struct Test {
+            foo: u8
+        }
+    """)
```

Comment:
What do you think about filtering derive attributes in case of `cfg_attr`?
For example,
```rust
#[cfg_attr(windows, derive(Debu/*caret*/))]
#[cfg_attr(windows, derive(Debug))]
struct Test {
    foo: u8
}
```

In this case, obviously, we shouldn't suggest `Debug` but in common case I don't think we can take into account all possible combinations of options/features. So I suggest just take into account all impls according to current options/features.
If you agree, let's add such test to catch such case when cfg evaluator will take into account `cfg_attr`

Reply:
Added a test